app/scripts/views/complete_reset_password.js

```diff
@@ -135,7 +136,15 @@ function (_, BaseView, FormView, Template, PasswordMixin,
         })
         .then(function (result) {
           if (! (result && result.halt)) {
-            self.navigate('reset_password_complete');
+            // the user is definitively signed in here, otherwise this
+            // path would not be taken.
+            if (self.relier.isDirectAccess()) {
+              self.navigate('settings', {
+                success: t('Account verified')
```

Comment:
Huh, I didn't realize that. I'm going to keep the existing `var t = BaseView.t;` just to stay consistent with how we do it elsewhere.